src/main/java/frc/robot/RobotContainer.java

```diff
@@ -126,4 +137,12 @@ public Command getAutonomousCommand() {
     // An example command will be run in autonomous
     return null;
   }
+
+  public Command getChargeStationRoutine(){
+    return AutoRoutines.chargeStationRoutine(m_pickupItemCommand,m_dropItemCommand, m_driveMetersCommand,new TurnPID(m_driveTrain, -(m_photonVision.getTarget().getYaw())));
```

Comment:
Following the comment on `AutoRoutines.java`:

```suggestion
    return Commands.sequence(m_pickupItemCommand,m_dropItemCommand, m_driveMetersCommand,new TurnPID(m_driveTrain, -(m_photonVision.getTarget().getYaw())));
```

Will do the same thing